src/fsd/4-entities/upgrade/upgrades.service.ts

```
@@ -528,4 +529,12 @@ export class UpgradesService {
     static isValidUpgrade(upgrade: string): boolean {
         return Object.hasOwn(recipeDataByName, upgrade);
     }
+
+    private static getFarmableCharacters(): IMaterialFull['id'][] {
+        const chars: IMaterialFull['id'][] = [];
+        return Object.values(this.recipeDataFull).reduce((acc, upgrade) => {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)